serverless/containers/index.mdx

```diff
@@ -2,7 +2,93 @@
 meta:
   title: Serverless Containers Documentation
   description: Serverless Containers Documentation
```

Comment:
add the description we agreed upon:
Dive into Scaleway <product> with our quickstart guides, how-tos, tutorials and more.

Reply:
Dive into Scaleway <product> with our quickstart guides, how-tos, tutorials and more.